catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/SierraItemsTest.scala

```diff
@@ -16,41 +15,31 @@ class SierraItemsTest extends FunSpec with Matchers with SierraDataUtil {
 
   describe("extractItemData") {
     it("parses instances of SierraItemData") {
-      val item1 = createSierraItemData
-      val item2 = createSierraItemData
+      val itemData = (1 to 2).map { _ => createSierraItemData }
+      val itemRecords = itemData.map { createSierraItemRecordWith }.toList
```

Comment:
`createSierraItemRecordWith` looks a bit odd here is it with defaults, `createSierraItemRecord`?

Reply:
No, we’re passing in the instances of SierraItemData. I’ll tweak to clarify.